lightning/src/ln/channel.rs

```diff
@@ -1876,7 +1872,8 @@ impl<SP: Deref> ChannelContext<SP> where SP::Target: SignerProvider  {
 		if let Some(feerate) = outbound_feerate_update {
```

Comment:
In the description above it is written:

```
// may, at any point, increase _by_ at least 10 sat/vB (i.e 2530 sat/kWU) or 25%,
// whichever is higher.
```
which implies:

```rust
cmp::max(feerate_by_kw + 2530, feerate_plus_quarter)
```

Whereas, the line should read:

```diff
pub fn get_dust_buffer_feerate(&self, outbound_feerate_update: Option<u32>) -> u32 {
                // When calculating our exposure to dust HTLCs, we assume that the channel feerate
-               // may, at any point, increase by at least 10 sat/vB (i.e 2530 sat/kWU) or 25%,
+               // may, at any point, increase to at least 10 sat/vB (i.e 2530 sat/kWU) or by 25%,
                // whichever is higher. This ensures that we aren't suddenly exposed to significantly
```

Since we are touching this function in this PR. I believe we should also address this small but significant comment change.

Reply:
Cool!